coreos-ostree-importer/README.md

```diff
@@ -135,14 +135,11 @@ You'll have to update the body with new information you'd like to use. Then run:
 
 ## Fedora Messaging sender from the Fedora CoreOS Pipeline
 
-Included in this directory is a file (`send-ostree-import-request.py`)
-that is not used by the `coreos-ostree-importer`
-at all. It is used by the
+In the cosa repo, you will find a script called `fedmsg-fcos-pipeline-ostree-import-request`. It is used by the
 [Fedora CoreOS Pipeline](https://github.com/coreos/fedora-coreos-pipeline.git)
-to send the request to the importer. It made sense to co-locate the
-requester and importer in the same code repo/directory.
+to send the request to the importer.
 
-Here's how you might send a request using `fcos-pipeline-ostree-import-request.py`: 
+Here's how you might send a request using `fedmsg-fcos-pipeline-ostree-import-request`:
 
 
 ```
```

Comment:
Can we update the below example:

- new name of file
- use `buildfetch` rather than `buildprep`